onadata/apps/logger/tasks.py

```diff
@@ -22,6 +23,13 @@ def create_monthly_counters():
         SubmissionCounter.objects.create(user_id=user_id)
 
 
+@task()
+def create_xform_monthly_counters():
+    xforms = XForm.objects.all()
+    for xform in xforms.iterator():
+        XFormSubmissionCounter.create(user_id=xform.user.id, xform=xform)
```

Comment:
seems to be dead code?

also, `create_monthly_counters()` seems to be used only by unit tests